### PR TITLE
Add 11.16.2020 update

### PR DESCRIPTION
**Pull request by jeremyrickard**
Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

#### What type of PR is this:
/kind documentation
/area release-team

#### What this PR does / why we need it:

This adds the markdown notes from 11/16/2020 to the meeting-notes directory

/hold for RT review
/cc @palnabarun @savitharaghunathan @hasheddan

### CODE REVIEW

File: releases/release-1.20/meeting-updates/2020-11-16.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# Meeting Updates 2020-11-16
+
+Cutoff Time: 2020-11-16 10:30:00 PST (Times in a few other timezones as well)
+Repository sync responsibility: Jeremy / @jerickar
+
+## Enhancements
+
+Author: Name / Slack Handle
+Current Status: Freen
```
Comment (hasheddan) [inline releases/release-1.20/meeting-updates/2020-11-16.md:9]: nitpick:
```suggestion
Current Status: Green
```

Comment (jeremyrickard) [reply releases/release-1.20/meeting-updates/2020-11-16.md:9]: yeah I didn't proof read, I just cut/paste. I don't like a formatting thing fixing that up too.

**Review by hasheddan — APPROVED** [review]
/lgtm

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Review by palnabarun — APPROVED** [review]

**Pull request merged**